package.json

```diff
@@ -59,7 +59,7 @@
     "electron-log": "^4.2.4",
     "electron-serve": "^1.0.0",
     "electron-updater": "^4.3.4",
-    "expensify-common": "git+https://github.com/Expensify/expensify-common.git#2e5cff552cf132da90a3fb9756e6b4fb6ae7b40c",
+    "expensify-common": "^1.0.1",
```

Comment:
We need to revert this. Sorry @jasperhuangg the SO instructions you followed are outdated.

Reply:
Hey @marcaaron, can you add context on why this is being reverted? Just curious to see what broke.

Reply:
We shouldn't be publishing `expensify-common` to npm. The advice in the SO is outdated (my bad). We must reference the commit hash instead like we have been doing for every other `expensify-common` update.